### PR TITLE
feat(home): introduce the Support section

### PR DESCRIPTION
**Pull request by im-klau**
## Name PR according scheme: type of changes (feat/fix/refactor etc.) + description
feat(home): introduce the Support section

## Jira issue

JIRA issue: [FSU-115](https://klau.atlassian.net/browse/FSU-115)

## Describe your changes
This PR introduces the Support section for the Home page.
The icons/illustrations probably will be replaced with the new/custom ones in the future so you can omit them while reviewing :D 

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/HomePage/components/Support/Support.styled.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  margin-top: 20px;
+  margin-bottom: 60px;
+`
+
+export const Content = styled.div`
+  position: relative;
+  z-index: 1;
+  max-width: 560px;
+  padding-top: 110px;
+  padding-bottom: 110px;
```
Comment (AmandaKr) [inline packages/frontend/app/templates/HomePage/components/Support/Support.styled.js:28]:
```suggestion
padding: 110px 0;
```

File: packages/frontend/app/templates/HomePage/components/Support/Support.styled.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+export const List = styled.ul`
+  max-width: 360px;
+  list-style: none;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 40px;
+  margin-top: 20px;
+  margin-bottom: 60px;
```
Comment (AmandaKr) [inline packages/frontend/app/templates/HomePage/components/Support/Support.styled.js:20]:
```suggestion
margin:20px 0 60px;
```

**Comment by nata-kab** [comment]
Great job 🚀  One thing just left - there should be no space to the left side of the content 😉 🕵️

Screenshot 2023-03-17 at 13 14 52
Screenshot 2023-03-17 at 13 15 24

File: packages/frontend/app/templates/HomePage/components/Support/Support.styled.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+`
+
+export const Item = styled.li`
+  flex: 1;
+  text-align: center;
+`
+
+export const Icon = styled(Illustration)`
+  height: 80px;
+`
```
Comment (nata-kab) [inline packages/frontend/app/templates/HomePage/components/Support/Support.styled.js:60]: Great job 🚀 Just one thing just - there should be no space to the left side of the content 😉 🕵️
Screenshot 2023-03-17 at 13 14 52
Screenshot 2023-03-17 at 13 15 24

Comment (im-klau) [reply packages/frontend/app/templates/HomePage/components/Support/Support.styled.js:58]: I'm not sure if I understand you 🙈 what do you mean by the space? the white one? I'm trying to compare it to the design and it looks quite similar

Is is about desktop viewport?

Comment (nata-kab) [reply packages/frontend/app/templates/HomePage/components/Support/Support.styled.js:58]: Yes, on the left side of the desktop view, there is a visible part of the background image (margin or padding of the container is given). It looks strange and does not match the design of the desktop view.

screenshot-fundation

Comment (nata-kab) [reply packages/frontend/app/templates/HomePage/components/Support/Support.styled.js:58]: Sorry. It's my mistake. I had compared it to the wrong homepage view from Figma.🙈

**Review by nata-kab — APPROVED** [review]
Looks great now 🚀

File: packages/frontend/app/templates/HomePage/components/Support/Support.styled.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+export const List = styled.ul`
+  max-width: 360px;
+  list-style: none;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 40px;
+margin: 20px 0 60px;
+`
```
[the remark this answers is not in the document]
Comment (im-klau) [reply packages/frontend/app/templates/HomePage/components/Support/Support.styled.js:31]: @nata-kab I've updated this layout, let me know if that works now!
and thank you! 🤗

**Review by paulgrym — COMMENTED** [review]
"Aktualnie poszukujemy" text should be underlined on mobile
image

Component text should have #0C0C0C color

**Comment by im-klau** [comment]
> "Aktualnie poszukujemy" text should be underlined on mobile image
> 
> Component text should have #0C0C0C color

We're unable and probably shouldn't override the Typography component to use additional underline on mobile viewport as it's not defined in our Design System. I asked Dorota to leave the heading without the underline for now 🙈 
We'll revisit this after launching v1.

However, I appreciate the thorough review, thank you!! :D 🙏

File: packages/frontend/app/templates/HomePage/components/Support/Support.styled.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+export const Content = styled.div`
+  position: relative;
+  z-index: 1;
+  max-width: 560px;
+  padding: 110px 0;
+
+  ${({ theme }) => theme.breakpoints.mobileLg} {
+    padding-top: 60px;
+    padding-bottom: 60px;
```
Comment (AmandaKr) [inline packages/frontend/app/templates/HomePage/components/Support/Support.styled.js:41]:
```suggestion
padding: 60px 0;
```

**Review by AmandaKr — APPROVED** [review]
Great job 😍

**Review by paulgrym — APPROVED** [review]
🌟

**Review by Ebmili — APPROVED** [review]
Good to go ;)

**Pull request merged**